tools/downloader/topologies.py

```diff
@@ -0,0 +1,191 @@
+import sys
+import os
+import re
+import yaml
+import types
+import argparse
+
+from pathlib import Path
+
+import common
+
+_downloader = Path(__file__).parent / 'downloader.py'
+_precision_marker = '$precision'
+_cache = Path(os.getenv('OPENCV_OPEN_MODEL_ZOO_CACHE_DIR', 'models')).resolve()
+
+
+for _topology in common.load_topologies(argparse.Namespace(config=None)):
+    _name = _topology.name.replace('-', '_').replace('.', '_')
+    _files = _topology.files
+    _download_dir = _cache / _topology.subdirectory
+    _mo_args = {}
+    if _topology.mo_args:
+        for _arg in _topology.mo_args:
+            _tokens = _arg.split('=')
+            _mo_args[_tokens[0]] = _tokens[1] if len(_tokens) == 2 else None
+
+    # Resolve weights and text files names.
+    assert(len(_files) > 0)
+
+    if len(_files) > 2:
```

Comment:
This condition is definitely wrong:

1. There are some DLDT format models that have only 2 files (the `INT1` ones).
2. While I don't think there are currently any models in any other format that have more than 2 files, there's no reason why there couldn't be in the future. For example, TensorFlow models can comprise at least 3 files (metagraph, checkpoint, pipeline config), and there's no telling what future formats could be like.

Reply:
Thanks! Removed this condition and replaced onto `_topology.framework == 'dldt'`.